lib/internal/bootstrap/node.js

```diff
@@ -125,6 +125,10 @@ if (!config.noBrowserGlobals) {
   // https://url.spec.whatwg.org/#urlsearchparams
   exposeInterface(global, 'URLSearchParams', URLSearchParams);
 
+  const { performance } = require('perf_hooks');
+  // https://www.w3.org/TR/performance-timeline/
```

Comment:
```suggestion
  // https://www.w3.org/TR/performance-timeline/
  // https://www.w3.org/TR/hr-time-2/
```

Reply:
We also implement https://www.w3.org/TR/user-timing/